gitter/dicttagger.py

```diff
@@ -0,0 +1,66 @@
+import yaml
+
+
+class DictionaryTagger(object):
+    def __init__(self, dictionary_paths):
+        files = [open(path, 'r') for path in dictionary_paths]
```

Comment:
It's unnecessary to load all files into memory together. Also, it's better to use context manager when reading from the file IMO.